package.json

```diff
@@ -1,69 +1,53 @@
-    "eslint-plugin-jest-dom": "^4.0.3",
-    "eslint-plugin-testing-library": "^5.9.1",
-    "eslint-plugin-unused-imports": "^2.0.0",
-    "gulp": "^4.0.2",
-    "gulp-babel": "^8.0.0",
-    "gulp-dart-sass": "^1.0.2",
-    "gulp-rename": "^2.0.0",
-    "gulp-uglify": "^3.0.2",
-    "jest": "^29.4.1",
-    "jest-axe": "^7.0.0",
-    "jest-environment-jsdom": "^29.4.1",
-    "jsdom": "^21.1.0",
-    "lerna": "^6.4.1",
-    "msw": "^1.0.0",
-    "prettier": "2.8.3",
-    "puppeteer": "^19.5.2",
-    "reg-keygen-git-hash-plugin": "^0.12.1",
-    "reg-notify-github-with-api-plugin": "^0.12.1",
-    "reg-publish-s3-plugin": "^0.12.1",
-    "reg-suit": "^0.12.1",
-    "rollup": "^3.12.0",
-    "stylelint": "^14.16.1",
-    "stylelint-config-prettier": "^9.0.4",
-    "stylelint-config-prettier-scss": "^0.0.1",
-    "stylelint-config-standard": "^29.0.0",
-    "stylelint-config-standard-scss": "^6.1.0",
-    "stylelint-scss": "^4.3.0",
-    "vinyl-buffer": "^1.0.1",
-    "vinyl-source-stream": "^2.0.0",
-    "whatwg-fetch": "^3.6.2"
+  "description": "The University of Queensland Design System",
+  "keywords": [
+    "uqds",
+    "uq",
+    "University of Queensland",
+    "design system",
+    "scss",
+    "sass",
+    "css",
+    "js"
+  ],
+  "homepage": "https://design-system.uq.edu.au",
+  "bugs": {
+    "url": "https://github.com/uq-its-ss/design-system/issues"
   },
+  "repository": "https://github.com/uq-its-ss/design-system/",
+  "author": "",
+  "license": "ISC",
+  "workspaces": [
+    "packages/*",
+    "storybook-html"
+  ],
+  "main": "index.html",
   "scripts": {
-    "format": "prettier --write \"**/*.{css,scss,js,jsx,json,md,mdx}\"",
-    "lint": "prettier --check \"**/*.{css,scss,js,jsx,json,md,mdx}\" && npm run lint-js && npm run lint-css",
-    "lint-js": "eslint --ext .js,.jsx .",
-    "lint-css": "stylelint \"**/*.{css,scss}\"",
-    "lint-fix": "npm run format && npm run lint-js -- --fix && npm run lint-css -- --fix",
-    "test": "jest",
-    "update-screenshots": "node scripts/screenshot.js",
-    "reg-suit": "reg-suit run",
-    "serve-screenshots": "http-server dist -i -p 6006"
+    "dev": "vite",
+    "build": "vite build",
+    "preview": "vite preview",
+    "format": "npx prettier . --write",
+    "lint": "npx prettier . --check && npm run lint-js", 
+    "lint-js": "npx eslint --ext .js,.jsx .",
+    "lint-fix": "npx run format && npx run lint-js -- --fix "
+  },
+  "devDependencies": {
+    "eslint": "^8.56.0",
+    "eslint-config-airbnb": "^19.0.4",
+    "eslint-config-prettier": "^9.1.0",
+    "eslint-plugin-import": "^2.29.1",
+    "eslint-plugin-jsx-a11y": "^6.8.0",
+    "eslint-plugin-react": "^7.33.2",
+    "eslint-plugin-react-hooks": "^4.6.0",
+    "lerna": "^6.5.1",
+    "prettier": "^3.1.1",
+    "vite": "^5.0.8"
   },
-  "browserslist": [
-    "last 4 chrome versions",
-    "last 4 firefox versions",
-    "safari > 14",
-    "ios_saf > 12"
-  ]
+  "engines": {
+    "node": ">=20.10.0",
```

Comment:
I was able to bump all the other uq repos to 18 - I did try 20 but a bunch of workflow stuff was failing so I gave up on that early on. I'm not sure if there is value in having everything on the same version or not.